File: Nested Logic/Day26.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
         Scanner N = new Scanner(System.in);
         int dayReturned = N.nextInt(); 
         int monthReturned = N.nextInt(); 
         int yearReturned = N.nextInt(); 

         int dayExpected  = N.nextInt(); 
         int monthExpected  = N.nextInt(); 
         int yearExpected  = N.nextInt(); 

         int yearLate=yearReturned-yearExpected;
         int monthLate=monthReturned-monthExpected;
         int dayLate=dayReturned-dayExpected;

         if(yearLate>0) System.out.println(10000);
         else{
             if(yearLate==0){
                if(monthLate>0) System.out.println(500*monthLate);           
                else if(monthLate==0&&dayLate>0) System.out.println(15*dayLate);
                else System.out.println(0);
             }
            else System.out.println(0);
         }
         }
}

File: Nested Logic/Day26.py
returned = str(input()).split(" ")
day_returned = int(returned[0])
month_returned = int(returned[1])
year_returned = int(returned[2])

expected = str(input()).split(" ")
day_expected = int(expected[0])
month_expected = int(expected[1])
year_expected = int(expected[2])    

year_late=year_returned-year_expected;
month_late=month_returned-month_expected;
day_late=day_returned-day_expected;

if(year_late>0): 
    print(10000)
else:
    if(year_late==0):
        if(month_late>0):
            print(500*month_late)          
        elif(month_late==0 or day_late>0): 
            print(15*day_late)
        else:
            print(0)
    else: 
         print(0)
         

